address/book.go

```diff
@@ -264,6 +265,25 @@ func (b *Book) QueryAssetInfo(ctx context.Context,
 	return assetGroup, nil
 }
 
+// SyncAssetGroup attempts to enable asset sync for the given asset group, then
+// perform an initial sync with the federation for that group.
+func (b *Book) SyncAssetGroup(ctx context.Context,
```

Comment:
Alternatively, we can have `QueryAssetInfo` accept an `asset.Specifier`.

Reply:
The way the function is setup, if you call it twice, only the first time will it actually try to fetch data.

Reply:
But `QueryAssetInfo` does the opposite of what we want: Given an asset ID it finds out what the group key is.
But what we need for this use case is: Given a group key, what asset IDs does that group contain?